replica/client/client.go

```diff
@@ -255,7 +256,16 @@ func (c *ReplicaClient) SendFile(from, host string, port int) error {
 				start = 1 * time.Second
 			}
 		case 0:
-			return nil
+			/*
+			* During sync process, degraded replica receives exitCode as success
```

Comment:
if healthy replica restarts, shouldn't the get call fail? As the new process will not have the process map for the PID?

Reply:
As per the logs pasted in PR, there seems to be a corner case where API server is still alive, and responded success.
With the fix, trying to verify in this case will fail, and, it marks SYNC process as failed.

Reply:
how are we sure that after sleep we will get the fail next time? That server could be up for that long also isn't it?

Reply:
Yes, there are edge cases possible.

Reply:
but, majority cases will be solved with this, unless re-design is made for sync process.